doc/version_history.rst

```diff
@@ -6,6 +6,19 @@
 Version History
 ###############
 
+v0.7.4
+======
+
+* Add wait_time class variable for mocking of timeouts.
+* Add pre-commit config file.
+* ``setup.cfg``: specify asyncio_mode=auto.
+
+Requires:
+
+* ts_tcpip 0.3
```

Comment:
Please consider 0.3.7 (which fixes monitoring) or 0.4 (not needed, just a rounder number)

Reply:
Done